package.json

```diff
@@ -22,7 +22,8 @@
         "nodegit": "^0.27.0",
         "picomatch": "^2.3.0",
         "tsconfig-paths": "^3.10.1",
-        "typescript": "^4.0.3"
+        "typescript": "^4.0.3",
+        "strip-json-comments": "^3.1.1"
```

Comment:
This breaks the alphabetic sorting of deps. Probably worth moving up.

Reply:
Wow, I'm bad at alphabetizing. Fixed.